rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -384,222 +380,45 @@ namespace rocal{
         m.def("GetIntValue",&rocalGetIntValue);
         m.def("GetFloatValue",&rocalGetFloatValue);
         // rocal_api_data_transfer.h
-        m.def("rocalCopyToOutput",&wrapper_copy_to_output);
+        m.def("rocalCopyToOutput",&wrapper);
```

Comment:
can the name be changed back to "wrapper_copy_to_output"?

Reply:
done